homework/remove-vowels/vowel.hpp

```diff
@@ -0,0 +1,7 @@
+#pragma once
+#include <vector>
+#include <string>
+#include <iostream>
+#include <algorithm>
```

Comment:
It's best to keep header files light, try to include only what is necessary. `iostream` and `algorithm` can be moved to .cpp file